File: Source code/src/main/java/bom/OrderDetail.java
package bom;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class OrderDetail {
	
	private int id;
	
	private Date dateCreated;
	
	private Product product;
	
	private Order order;
	
	private Bot bot;
	
	private Cluster cluster;
	
	private Date dateStart;
}

File: Source code/classes/production/Frontend/shared/models/order-detail.ts
import {Product} from './product';
import {Bot} from './bot';
import {Cluster} from './cluster';
import {Order} from './order';

export interface OrderDetail {
  id: number;
  product: Product;
  order: Order;
  bot: Bot;
  cluster: Cluster;
  dateCreated: Date;
  dateStart: Date;
}

